DEVELOPMENT.md

```diff
@@ -84,8 +145,29 @@ ROOT_MAIL_DOMAIN=... \
 
 #### Create a pull request
 
-Creating a PR will trigger the [build job](tests/build.yaml) and run the test suite (if PR creator is in the allow-list).
+Creating a PR will trigger the [build job](tests/build.yaml) and run the test suite (if PR creator is in the `AllowedGitHubAccountIDs` as mentioned above).
+
+## Releasing
+
+We are using the [semantic-release-action](https://github.com/cycjimmy/semantic-release-action), so the plugin determines which version it will cut based on the semantic commit messages (see the format [here](https://github.com/semantic-release/semantic-release)) from the last release until now. 
```

Comment:
please add that we are using conventional commits https://www.conventionalcommits.org/

Reply:
*was decided in joint session

Reply:
yes will add the link